src/main/java/de/paul2708/memory/game/Game.java

```diff
@@ -200,7 +201,7 @@ private void createInventory() {
 
         for (int i = 0; i < inventory.getSize(); i++) {
             ItemStack item = inventory.getItem(i);
-            if (item == null || item.getType() == null || item.getType() == Material.AIR) {
+            if (item == null || item.getType() == Material.AIR) {
```

Comment:
I'm not quite sure if it's even possible, but since version 1.16, there are multiple types of `AIR` (e.g. [`CAVE_AIR`](https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/Material.html#CAVE_AIR)). So maybe `item.getType().isAir()` would fit better.

Reply:
I just removed `item.getType() == null` due to `@NotNullable` returnment of `getType`